python/.ci/Jenkinsfile

```diff
@@ -430,6 +430,58 @@ pipeline {
                             }
+                            stage('PUB/SUB LITE TO BIGTABLE') {
+                                steps{
+                                    sh '''
+                                    
+                                        python3.8 -m pip install --user virtualenv    
+    
+                                        python3.8 -m venv env
+    
+                                        source env/bin/activate
+                                        
+                                        export GCS_STAGING_LOCATION="gs://python-dataproc-templates-temp"
+                                        export JARS="gs://spark-lib/pubsublite/pubsublite-spark-sql-streaming-LATEST-with-dependencies.jar"
+                                        export SKIP_BUILD=true
+
+                                        gcloud pubsub lite-topics publish psltobt \
+                                        --location=us-west1 \
+                                        --message=\'\'\'
+                                        { 
+                                            "rowkey":"rk1",
+                                            "columns": [
+                                                {
+                                                    "columnfamily":"place",
+                                                    "columnname":"city",
+                                                    "columnvalue":"Bangalore"
+                                                },
+                                                {
+                                                    "columnfamily":"place",
+                                                    "columnname":"state",
+                                                    "columnvalue":"Karnataka"
+                                                },
+                                                {
+                                                    "columnfamily":"date",
+                                                    "columnname":"year",
+                                                    "columnvalue":"2023"
+                                                }
+                                            ] 
+                                        }
+                                        \'\'\'
+    
+                                        cd python
+
+                                        ./bin/start.sh \
+                                        -- --template=PUBSUBLITETOBIGTABLE \
+                                        --pubsublite.bigtable.subscription.path=projects/$GCP_PROJECT/locations/$REGION/subscriptions/psltobt-sub \
+                                        --pubsublite.bigtable.streaming.checkpoint.location="gs://dataproc-templates/integration-testing/psltobt/checkpoint" \
+                                        --pubsublite.bigtable.output.project=$GCP_PROJECT \
+                                        --pubsublite.bigtable.output.instance="tanyawarrier-bt-1" \
```

Comment:
Please avoid using your ldaps for integration testing. We are leveraging an existing bigtable instance for other java templates. You can use the same environment variable: $ENV_TEST_BIGTABLE_INSTANCE. Please rename the output table name as well.